csharp/ql/src/semmle/code/cil/Instructions.qll

```diff
@@ -6,85 +6,98 @@ private import CIL
 private import semmle.code.dotnet.Variable as DotNet
```

Comment:
Could we perhaps add a link to something like https://en.wikipedia.org/wiki/List_of_CIL_instructions to the file QLDoc?